externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/ExecuteStatement.scala

```diff
@@ -124,6 +126,10 @@ class ExecuteStatement(
             rejected)
           setOperationException(ke)
           throw ke
+        case NonFatal(e) =>
```

Comment:
Is this a bug you try to fix here？

Reply:
refered to

https://github.com/apache/spark/blob/cb30683b65714bd1c1ae15d31d1411aeb66cef12/sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala#L260